### ISSUE TITLE
can you make this into a DLL please? :)

### ISSUE THREAD

**Issue by Bluscream** [FolfyBlue/GetFreeGames#2]

**Comment by NoNameTrademark** [comment]
@FolfyBlue  Yes ! Can you please. And how does it work? I do not understand. Thanks. :D

**Comment by JoshuaVandaele** [comment]
I'm leaving this open, but making something a DLL is something out of what I can do, it'd require rewriting this in a language that can be compiled as such. If someone else wants to do this, they're open to

**Comment by mezoo81** [comment]
@FolfyBlue How i can its work ? how can i addo this to ArchiSteamFarm ?

**Comment by NoNameTrademark** [comment]
@FolfyBlue how it works ? how i added to how i added to ArchiSteamFarm ?

**Comment by VarChar42** [comment]
I could port it to a plugin for ArchiSteamFarm. So it becomes a dll. But in my opinion this isn't necessary. A python script using the IPC api is a pretty elegant solution.
